data/hai/hai missions.txt

```diff
@@ -506,7 +506,8 @@ mission "Expanding Business [1]"
 	source "Greenwater"
 	destination "Allhome"
 	to offer
-		has "event: hai-human resolution announced"
+		"combat rating" > 100
+		random < 90
```

Comment:
```suggestion
		has "First Contact: Hai: offered"
		has "main plot completed"
```
These were the requirements before it was locked behind HR, I would just restore these.

Reply:
Actually, you might also want to restore the old `on offer` conversation from before HR, too.